### DISCUSSION TITLE
[A-L] (2023/24) Foglio 1 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#135] [Esercizi - Bertini]
![Es 4](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/239463da-3ec4-4bc6-8c76-22e21dbe3161)

**Comment by Elia-Belli** [comment]
Sia $S$ l'insieme di tutti gli studenti, $F$ l'insieme degli studenti che ha studiato francese e $T$ l'insieme degli studenti che ha studiato tedesco, sappiamo che:
$|S|=50$; $|F| = 25$; $|T| = 20$; $|F \cap T| = 5$; 
- **1)** Gli studenti che hanno studiato solo francese sono quelli che non hanno studiato anche tedesco $25-5 = 20$
- **2)** Gli studenti che hanno studiato solo trdesco sono quelli che non hanno studiato anche francese $20-5 = 15$
- **3)** Tra i restanti $50-20-15 = 15$ studenti, solo $5$ hanno studiato entrambe le lingue quindi $15-5 = 10$ non hanno studiato ne francese ne tedesco.

**Comment by CiottoloMaggico** [comment]
Analogo a @Elia-Belli
